code/controllers/subsystems/runechat.dm

```diff
@@ -0,0 +1,3 @@
+TIMER_SUBSYSTEM_DEF(runechat)
+	name = "Runechat"
```

Comment:
I know I'm harping on about this, but I feel like falling it 'floating chat' or something is more immediately obvious what it's for. As mentioned on Discord, calling it runechat just because runescape was the initial inspiration isn't particularly helpful from a maintenance and understanding perspective.